mlir/test/python/dialects/transform_extras.py

```diff
@@ -93,3 +118,73 @@ def test_match_ops_mixed(op: OpHandle):
     # CHECK-NEXT: %[[VAL_1:.*]] = transform.structured.match
     # CHECK-SAME:   ops{["scf.for", "linalg.matmul", "scf.forall"]} in %[[VAL_0]]
     # CHECK-SAME:     -> !transform.any_op
+
+
+# CHECK-LABEL: TEST: test_sequence_region
+@construct_and_print_in_module
+def test_sequence_region():
```

Comment:
These tests look more like integration tests to me as the transform script here seems to be meant to operate on the  generated payload IR.  Does it make sense to either:
- Move them to a different file and also run the TransformDialectInterpreter pass to check that the actually fit together and produce the correct output.
or:
- Only have the transform specific parts tested here

Reply:
Indeed in my "staging" repo I do run TransformInterpreter. Let me actually do both of these - remove the target func here but then add them both to wherever TransformInterpreter is tested.